tests/testthat/test-utilities-quantity.R

```diff
@@ -102,17 +102,21 @@ test_that("It can set multiple quantity values with units", {
 
 test_that("It throws an exception when setting values for a quantity that does not exist", {
   parameterPath <- "Organism|Liver|NOPE|Volume"
-  expect_that(setQuantityValuesByPath(parameterPath, 100, sim), throws_error())
+  expect_error(setQuantityValuesByPath(parameterPath, 100, sim))
+})
+
+test_that("It does not throw an exception when setting values for a quantity that does not exist with unit", {
+  parameterPath <- "Organism|Liver|NOPE|Volume"
+  expect_error(setQuantityValuesByPath(quantityPaths = parameterPath, values = 100, simulation = sim, units = "ml", stopIfNotFound = FALSE), NA)
```

Comment:
See above. 

The same for the two other instances below of `expect_error()`.

Reply:
Here we are actually testing that it does NOT throw an error :D

Reply:
Ah, I think, then this should be `expect_output()`.

Reply:
Changed to expect_output()

Reply:
`expect_output` !! I was wondering how to test that there is NO Error lol. THANK YOU @IndrajeetPatil . A quick search in SO showed that I need to put NA at the end. But this looked very weird to me and I was about to write an expect_no_error method lol (which would be easier to understand than expect_output IMO)